### PR TITLE
add test for ValidatePort

### PR DESCRIPTION
**Pull request by muyangren2**
…o for ValidatePort

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Add UT
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
 "NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go [hunk 1]
```diff
@@ -183,6 +185,30 @@ func TestValidateIPFromString(t *testing.T) {
 	}
 }
 
+func TestValidatePort(t *testing.T) {
+	var tests = []struct {
+		name     string
+		port     int32
+		expected bool
+	}{
+
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go:194]: remove empty line

File: cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go [hunk 2]
```diff
@@ -183,6 +185,30 @@ func TestValidateIPFromString(t *testing.T) {
...
+		{"negative number port", -1234, false},
+		{"zero number port", 0, false},
+		{"Minimum valid value port", 1, true},
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go:197]: use lowercase text consistently for test names or uppercase

File: cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go [hunk 3]
```diff
@@ -183,6 +185,30 @@ func TestValidateIPFromString(t *testing.T) {
...
+		{"valid value port", 300, true},
+		{"maximum valid value", 65535, true},
+		{">65535 Invalid value port", 65538, false},
+	}
+	for _, rt := range tests {
+		allErrs := ValidatePort(rt.port, nil)
+		//allErrs[0].Error()
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go:204]: remove commented line

File: cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go [hunk 4]
```diff
@@ -183,6 +185,30 @@ func TestValidateIPFromString(t *testing.T) {
...
+		if len(allErrs) > 0 {
+			find := strings.Contains(allErrs[0].Error(), "port number is not valid")
+			assert.Equal(t, rt.expected, !find, allErrs[0].Error())
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go:207]: can we just use regular compare instead of testify?

Comment (muyangren2) [reply cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go:207]: I will modify it according to your opinion，thank you

**Review by pacoxu — APPROVED** [review]
/lgtm
/priority backlog

**Review by neolit123 — COMMENTED** [review]
/approve

**Pull request merged**